cloud/jenkins/pgo_operator_eks.groovy

```diff
@@ -1,135 +1,303 @@
-                        fi
-
-                        if [ -n "${PGO_SCHEDULER_IMAGE}" ]; then
-                            export IMAGE_SCHEDULER=${PGO_SCHEDULER_IMAGE}
-                        else
-                            export IMAGE_SCHEDULER=perconalab/percona-postgresql-operator:${env.GIT_BRANCH}-pgo-scheduler
-                        fi
-
-                        if [ -n "${PGO_DEPLOYER_IMAGE}" ]; then
-                            export IMAGE_DEPLOYER=${PGO_DEPLOYER_IMAGE}
-                        else
-                            export IMAGE_DEPLOYER=perconalab/percona-postgresql-operator:${env.GIT_BRANCH}-pgo-deployer
-                        fi
-
+    
                         if [ -n "${PGO_PGBOUNCER_IMAGE}" ]; then
                             export IMAGE_PGBOUNCER=${PGO_PGBOUNCER_IMAGE}
                         fi
-
-                        if [ -n "${PGO_POSTGRES_HA_IMAGE}" ]; then
-                            export IMAGE_PG_HA=${PGO_POSTGRES_HA_IMAGE}
-                            export PG_VER=\$(echo \${IMAGE_PG_HA} | grep -Eo 'ppg[0-9]+'| sed 's/ppg//g')
+    
+                        if [ -n "${PGO_POSTGRES_IMAGE}" ]; then
+                            export IMAGE_POSTGRESQL=${PGO_POSTGRES_IMAGE}
+                            export PG_VER=\$(echo \${IMAGE_POSTGRESQL} | grep -Eo 'ppg[0-9]+'| sed 's/ppg//g')
                         fi
-
+    
                         if [ -n "${PGO_BACKREST_IMAGE}" ]; then
                             export IMAGE_BACKREST=${PGO_BACKREST_IMAGE}
                         fi
-
-                        if [ -n "${PGO_BACKREST_REPO_IMAGE}" ]; then
-                            export IMAGE_BACKREST_REPO=${PGO_BACKREST_REPO_IMAGE}
-                        fi
-
+    
                         if [ -n "${PGO_PGBADGER_IMAGE}" ]; then
                             export IMAGE_PGBADGER=${PGO_PGBADGER_IMAGE}
                         fi
-
+    
                         if [ -n "${PMM_SERVER_IMAGE_BASE}" ]; then
                             export IMAGE_PMM_SERVER_REPO=${PMM_SERVER_IMAGE_BASE}
                         fi
-
+    
                         if [ -n "${PMM_SERVER_IMAGE_TAG}" ]; then
                             export IMAGE_PMM_SERVER_TAG=${PMM_SERVER_IMAGE_TAG}
                         fi
-
+    
                         if [ -n "${PMM_CLIENT_IMAGE}" ]; then
                             export IMAGE_PMM=${PMM_CLIENT_IMAGE}
                         fi
-
-                        export PATH=/home/ec2-user/.local/bin:$PATH
+    
+                        export KUBECONFIG=/tmp/$CLUSTER_NAME-$clusterSuffix
+                        export PATH="$HOME/.krew/bin:$PATH"
                         source $HOME/google-cloud-sdk/path.bash.inc
-                        export KUBECONFIG=~/.kube/config
-
-                        ./e2e-tests/$TEST_NAME/run
-                    fi
-                """
+                        set -o pipefail
```

Comment:
Do we need to set this since we are not piping anything until the end of the script?